types/qrcode/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for qrcode 1.2
+// Type definitions for qrcode 1.3.0
```

Comment:
It should be `// Type definitions for qrcode 1.3` here

Reply:
Doing that now